test.js

```diff
@@ -27,3 +27,59 @@ it('should have a secDecimalDigits option', function () {
 	assert.strictEqual(prettyMs(33333, {secDecimalDigits: 0}), '33s');
 	assert.strictEqual(prettyMs(33333, {secDecimalDigits: 4}), '33.3330s');
 });
+
+it('should have a verbose option', function () {
+
+	function prettyMsVerbose (ms) {
```

Comment:
No it isn't: `function prettyMsVerbose(ms) {`. You only need a space after the `function` keyword.